addons/stock/models/stock_move.py

```diff
@@ -416,51 +416,86 @@ def onchange_date(self):
         if self.date_expected:
             self.date = self.date_expected
 
-    # TDE DECORATOR: remove that api.multi when action_confirm is migrated
-    @api.multi
+    def _key_assign_picking(self):
```

Comment:
On pourrait rajouter les commentaires # OF Modification OpenFire et Fin autour de ces modifs ?